beacon_node/network/src/sync/block_lookups/tests.rs

```diff
@@ -29,15 +31,44 @@ type T = Witness<ManualSlotClock, CachingEth1Backend<E>, E, MemoryStore<E>, Memo
 struct TestRig {
     beacon_processor_rx: mpsc::Receiver<WorkEvent<E>>,
     network_rx: mpsc::UnboundedReceiver<NetworkMessage<E>>,
+    network_rx_queue: Vec<NetworkMessage<E>>,
     rng: XorShiftRng,
     harness: BeaconChainHarness<T>,
+    sync_manager: SyncManager<T>,
+    beacon_processor: Arc<NetworkBeaconProcessor<T>>,
```

Comment:
Any reasons to have `beacon_processor` in the `TestRig`? I don't see it being used other than accessing `network_globals`.

I kept it out because the test rig should not need to directly interact with the `beacon_processor` directly in any way, otherwise it could interfere testing of the `SyncManager`.

Reply:
Good point! Removed beacon_processor for network_globals, and harness for fork_name